<launch>
  <group ns="tiago_dual">
  <arg name="use_knowrob" default="false"/>

  <node pkg="kdl_ik_service" type="start_ros_server.py" name="tiago_dual_kdl" output="screen" />

  <param name="robot_description"
         textfile="$(find pycram)/resources/robots/tiago_dual.urdf"/>

  <!-- Use Knowrob -->
  <group if="$(eval use_knowrob)">
      <include file="$(find knowrob)/launch/knowrob.launch"/>
  </group>

  </group>

</launch>
